src/app/components/media/ItemTitle.module.css

```diff
@@ -0,0 +1,48 @@
+.itemTitle {
+  display: flex;
+  flex: 0 0 100%;
+  gap: 8px;
+}
+
+.itemTitleInputField {
+  width: 100%;
+}
+
+.itemTitleMenuHeader {
+  color: var(--textSecondary);
+}
+
+.itemTitleOptionIcon {
```

Comment:
more of a suggestion than a necessary change, but you can avoid using the `!important` in these css statements by using the css loader global scope, such as `:global(.MuiListItemText-root) {...`

what that does is, "breaks" out of the css modules rewriting concept and just uses the plain classname so you can target things like material components, etc.